src/views/Profile/Profile.tsx

```diff
@@ -25,6 +29,24 @@ const Profile = () => {
     }
   }, [isLoading, isAuthenticated, user_profile, history]);
 
+  const handleResumeUploadReady = () => {
+    if (uploadRef.current.files.length !== 1 ||
+      (!uploadRef.current.files[0].name.endsWith(".pdf") &&
+      !uploadRef.current.files[0].name.endsWith(".doc") &&
+      !uploadRef.current.files[0].name.endsWith(".docx"))) return alert("Please make sure you upload a single file ending in .pdf, doc, or docx");
+    setUploadReady(true);
+  };
+  const handleResumeUpload = () => {
+    let extension = "pdf";
+    const file = uploadRef.current.files[0];
+    if (file.name.endsWith(".doc")) extension = "doc";
+    else if (file.name.endsWith(".docx")) extension = "docx";
+
+    const resumeRef = ref(storage, `${process.env.REACT_APP_GCP_RESUME_PATH}/${user_profile.profile?.net_id}.${extension}`);
```

Comment:
I'd recommend doing `user_profile.profile.sub` instead of `net_id`. Keep it consistent with the same unique key used for profiles in firestore. When downloading the resumes from cloud storage it'd be simple to substitute in names / net_ids / etc but programmatically safer to use `sub`. Also, you're guaranteed that every user has a `sub` but not every user might have a `net_id` since non-utd students that have portal accounts don't submit a net_id parameter on sign up.

Reply:
Will do. I thought accounts required a net_id, so I suppose I was misinformed. Will change it shortly.